### PR TITLE
complete 8-patch-votes-by-review-id

### PR DESCRIPTION
**Pull request by felcard**

### CODE REVIEW

File: __tests__/index.test.js
Comment (callumu21) [inline __tests__/index.test.js]: You should just be returning on a key of 'review' here, rather than 'updatedReviewVote'

File: __tests__/index.test.js
Comment (callumu21) [inline __tests__/index.test.js]: This shouldn't be a 404 error. Personally, I'd allow negative votes, but it depends how you want it to work on the front-end. Either way though, it's not a bad request. How about returning a vote set to 0 if the downvotes are more than the current total?

File: controllers/controllers.js
Comment (callumu21) [inline controllers/controllers.js]: This should be returned on a key of review

File: __tests__/index.test.js
Comment (callumu21) [inline __tests__/index.test.js:324]: You also need to test for a body that is missing inc_votes, or has it misspelled.

**Review by callumu21 — CHANGES_REQUESTED** [review]
A few changes needed before approval. Good start, though 👍

File: models/models.js
Comment (pkaodev) [inline models/models.js:60]: Be careful here.  You are checking if the object returned from the query has negative votes, and then sending back a new object where votes = 0, but the value in the database will still be negative.

There are a couple of things you can do to fix it (up to you):
-allow negative votes
-another query inside the if statement that 'if votes is negative, set votes to 0'
-change your initial query so that votes doesn't go below zero (trickiest)

**Comment by pkaodev** [comment]
See my comment, let me know if you want to discuss it.  Please submit another PR after resolving.

**Pull request merged**
